lib/gis_robot_suite/descriptive_metadata_builder.rb

```diff
@@ -58,18 +58,17 @@ def alternate_titles
 
     def event
       citation_nodes = data_id_node.xpath('gmd:citation/gmd:CI_Citation', NS)
-      publisher_nodes = citation_nodes.xpath("gmd:citedResponsibleParty/gmd:CI_ResponsibleParty/gmd:role/gmd:CI_RoleCode[text()='publisher']", NS)
-      raise "Publisher missing for #{bare_druid}." unless publisher_nodes.any?
 
       [].tap do |events|
-        events << extract_publishers(publisher_nodes)
-                  .merge(extract_pubdate(publisher_nodes.first)) # use first publisher node to navigate to date elements
-                  .merge(extract_edition(publisher_nodes.first)) # use first publisher node to navigate to edition elements
+        events << extract_pubdate(citation_nodes)
```

Comment:
Since a publisher is not required, merge instead in into pubdate, a required field.